composer.lock

```diff
@@ -2517,16 +2516,16 @@
         },
         {
             "name": "behat/mink-selenium2-driver",
-            "version": "1.3.x-dev",
+            "version": "dev-master",
```

Comment:
Kinda surprised that is needed. Whats causing this?

Reply:
Good catch. I accidentally committed the wrong lock file. Pushed another commit that fixes it.